build/go.mk

```diff
@@ -107,6 +107,10 @@ lint: lint-deps ## (Lint) Run the linters (golangci-lint and protolint)
 	$(tools/golangci-lint) run --timeout 2m ./...
 	$(tools/protolint) lint rpc
 
+.PHONY: lint-chart
+lint-chart:
+	helm lint charts/telepresence --set isCI=true
+
```

Comment:
Is there a good reason this isn't part of the main `make lint`?

Reply:
Not sure if a _good_ reason, but in case somebody wanted to _only_ lint the chart or _only_ do the go lint, having them separate would be easier... but the commands themselves aren't that complicated so maybe it's cleaner to combine them and then someone can look the commands up if they only want to do one or the other.